### PR TITLE
fix : [NSApp mainMenu] return is nil.  It need add notification NSApp…

### PR DESCRIPTION
**Pull request by JxbSir**
…licationDidFinishLaunchingNotification .

### CODE REVIEW

File: Project Templates/Application Plug-in/Xcode Plugin.xctemplate/___PACKAGENAME___.m [hunk 1]
```diff
@@ -25,21 +25,29 @@ - (id)initWithBundle:(NSBundle *)plugin
...
+    
+    // Sample Menu Item:
+    NSMenuItem *menuItem = [[NSApp mainMenu] itemWithTitle:@"Edit"];
+    if (menuItem) {
+        [[menuItem submenu] addItem:[NSMenuItem separatorItem]];
+        NSMenuItem *actionMenuItem = [[NSMenuItem alloc] initWithTitle:@"Do Action" action:@selector(doMenuAction) keyEquivalent:@""];
+        //[actionMenuItem setKeyEquivalentModifierMask:NSAlphaShiftKeyMask | NSControlKeyMask];
+        [actionMenuItem setTarget:self];
+        [[menuItem submenu] addItem:actionMenuItem];
+    }
```
Comment (kattrali) [inline Project Templates/Application Plug-in/Xcode Plugin.xctemplate/___PACKAGENAME___.m]: I think if we `removeObserver` here, this fix should be perfect.

**Comment by JxbSir** [comment]
What do you mean?

**Comment by kattrali** [comment]
Once the notification is received, the observer should be removed.

**Comment by JxbSir** [comment]
OK, I've removed Observer

**Comment by kattrali** [comment]
Looks good, thanks for the contribution! :)

**Pull request merged**